akka-cluster/src/multi-jvm/scala/akka/cluster/RemoteFeaturesWithClusterSpec.scala

```diff
@@ -30,7 +30,7 @@ class ClusterRemoteFeaturesConfig(artery: Boolean) extends MultiNodeConfig {
     ConfigFactory.parseString(s"""
       akka.remote.log-remote-lifecycle-events = off
       akka.remote.artery.enabled = $artery
-      akka.remote.artery.canonical.port = ${MultiNodeSpec.tcpPort}
+      akka.remote.artery.canonical.port = ${MultiNodeSpec.selfPort}
```

Comment:
Actually, this was correct. It should always use the selfPort. In most tests this will be the tcpPort. For the multinode aureon udp tests, the selfPort will be the udpPort. 

When not running on artery, the settings won't be used and we will use `akka.remote.classic.netty.tcp.port` instead which is not always using the tcpPort.

That gives us the following schema:

* multi-node tests: classic with tcpPort
* multi-node tests: artery with tcpPort 
* multi-node + aeron-udp tests: artery with udpPort
* multi-node + aeron-udp tests: classic with tcpPort

We need to improve the last one since we are testing it twice and it's confusion. We should disable the classic variant when running the aeron-udp tests.